var/SymfonyRequirements.php

```diff
@@ -512,7 +512,7 @@ function_exists('simplexml_import_dom'),
         if (extension_loaded('suhosin')) {
             $this->addPhpIniRequirement(
                 'suhosin.executor.include.whitelist',
-                function ($cfgValue) { return false !== stripos($cfgValue, "phar"); },
+                create_function('$cfgValue', 'return false !== stripos($cfgValue, "phar");'),
```

Comment:
Watch out #9409

Reply:
Maybe we could put `var/SymfonyRequirements.php` in `.gitignore` ?

Reply:
Like `./bin/console` this one needs to be committed. Was a little fix, waiting for the day we'll update Symfony =)

Reply:
This is generated automatically when using composer. Why is that?

Reply:
From Symfony: https://github.com/symfony/symfony-standard/issues/669#issuecomment-45850914
And for bin/console: https://github.com/github/gitignore/blob/master/Symfony.gitignore#L30
We need to keep @mickaelandrieu's changes just to be compliant with PHP7.2, otherwise, you will cry when trying to install PrestaShop with the latest version.

Reply:
Maybe related to the php version which is used when running composer?

Reply:
No, related to the fact that Symfony must be compatible with php 5.3 :/